lms/templates/certificates/_accomplishment-banner.html

```diff
@@ -85,10 +85,10 @@ <h3 class="sr-only">${_("Print or share your certificate:")}</h3>
                     %endif
 
                     %if linked_in_url:
-                    <a class="action action-linkedin-profile btn btn-overlay btn-small icon-only" id="action-share-linkedin" target="_blank" href="${linked_in_url}" title="${_('Add to LinkedIn Profile')}" data-course-id="${course_id}" data-certificate-mode="${course_mode}">
+                    <button class="action action-linkedin-profile btn btn-overlay btn-small icon-only" id="action-share-linkedin" onclick="window.open('${linked_in_url}');" title="${_('Add to LinkedIn Profile')}" data-course-id="${course_id}" data-certificate-mode="${course_mode}">
```

Comment:
Thanks for the feedback @talbs -- @andy-armstrong do you have a recommendation on how @asadiqbal08 might go about moving the onclick handler into the JS layer as a part of this PR?